ipatests/test_integration/test_caless.py

```diff
@@ -1513,3 +1514,42 @@ def test_server_replica_install_pkinit(self):
                                       pkinit_pin=_DEFAULT)
         assert result.returncode == 0
         self.verify_installation()
+
+
+class TestServerReplicaCALessToCAFull(CALessBase):
+    """
+    Test server and replica caless to cafull scenario:
+    Master (caless) / replica (caless) >> master (ca) / replica (ca)
+    """
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        super(TestServerReplicaCALessToCAFull, cls).install(mh)
+
+        cls.create_pkcs12('ca1/server')
+        cls.prepare_cacert('ca1')
+
+        master = cls.install_server()
+        assert master.returncode == 0
+
+    def test_server_ipa_ca_install(self):
+        """Install CA on master"""
+
+        self.create_pkcs12('ca1/replica', filename='replica.p12')
+
+        replica = self.prepare_replica()
```

Comment:
I'd like to see replica installation in a separate method, because it's not related to installing ca on the master.